conf/test.config

```diff
@@ -113,3 +117,67 @@ if (System.getenv('PROFILE')) {
         shifter.enabled                          = false
     }
 }
+
+
+profiles {
+    conda {
+        conda.enabled           = true
+        conda.channels          = ['conda-forge', 'bioconda']
+        apptainer.enabled       = false
+    }
+    mamba {
+        conda.enabled           = true
+        conda.useMamba          = true
+    }
+    docker {
+        docker.enabled          = true
+        docker.runOptions       = '-u $(id -u):$(id -g) --platform=linux/amd64'
+    }
+    docker_self_hosted {
+        docker.enabled          = true
+        docker.fixOwnership     = true
+        docker.runOptions       = '--platform=linux/amd64'
+    }
+    arm {
+        docker.runOptions       = '-u $(id -u):$(id -g) --platform=linux/amd64'
+    }
+    singularity {
+        singularity.enabled     = true
+        singularity.autoMounts  = true
+    }
+    podman {
+        podman.enabled          = true
+        podman.runOptions       = "--runtime crun --platform linux/x86_64 --systemd=always"
+    }
+    shifter {
+        shifter.enabled         = true
+    }
+    charliecloud {
+        charliecloud.enabled    = true
+    }
+    apptainer {
+        apptainer.enabled       = true
+        apptainer.autoMounts    = true
+    }
+    wave {
+        apptainer.ociAutoPull   = true
+        singularity.ociAutoPull = true
+        wave.enabled            = true
+        wave.freeze             = true
+        wave.strategy           = 'conda,container'
+    }
+    gitpod {
+        executor.name           = 'local'
+        executor.cpus           = 4
+        executor.memory         = 8.GB
+    }
+    gpu {
+        docker.runOptions       = '-u $(id -u):$(id -g) --gpus all'
+        apptainer.runOptions    = '--nv'
+        singularity.runOptions  = '--nv'
+    }
+}
+
+docker.registry      = 'quay.io'
+podman.registry      = 'quay.io'
+singularity.registry = 'quay.io'
```

Comment:
Is this needed?

Reply:
yeah, I'd put this in the main nextflow.config, similarly to the arm and other profiles